src/render/portable/mod.rs

```diff
@@ -362,6 +386,38 @@ fn get_linked_tables(table: &str, specs: &TablesSpec) -> Result<LinkedTable> {
     Ok(result)
 }
 
+/// Renders tick plots for given csv column
+fn render_tick_plot(
+    title: &str,
+    csv_path: &Path,
+    separator: char,
+    header_rows: usize,
+    tick_plot: &TickPlot,
+) -> Result<String> {
+    let mut reader = csv::ReaderBuilder::new()
+        .delimiter(separator as u8)
+        .from_path(csv_path)?;
+
+    let column_index = reader
+        .headers()
+        .map(|s| s.iter().position(|t| t == title).unwrap())?;
+
+    let (min, max) = get_min_max(csv_path, separator, column_index, header_rows)?;
```

Comment:
Just a thought: can we get min and max for free from the initial run through the table, without having to run through it again?

Reply:
The initial run items the records in groups. We would also need to check the specs before the iter to see what column needs to have the minimum and maximum determined. I guess it would be doable but probably not in a very pretty way.